.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,20 @@
-<!--
-Thank you for your interest in and contributing to Kibana! There
-are a few simple things to check before submitting your pull request
-that can help with the review process. You should delete these items
-from your submission, but they are here to help bring them to your
-attention.
-
-- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
-- Have you followed the [contributor guidelines](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md)?
-- If submitting code, have you included unit tests that cover the changes?
-- If submitting code, have you tested and built your code locally prior to submission with `yarn test && yarn build`?
-- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
--->
\ No newline at end of file
+### Summary
+
+Summarize your PR. If it involves visual changes include a screenshot or gif.
+
+### Checklist
+
+Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.
+
+- [ ] This was checked for cross-browser compatability, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)
+- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
+- [ ] Any code added follows our [style guides](https://github.com/elastic/kibana/blob/master/style_guides/)
```

Comment:
I am a bit worried about that point. Of course we want people to follow the styleguide, but we currently have 9 different style guides with around 2200 lines. Even when reading all of them, it's hard to keep all of that in mind for a longer time. Imho before we should make people sign off, that they didn't violate any style guide, we should try to strip down our style guides a lot, which imho would include removing everything from the style guides that we can check automatically (e.g. spaces over tabs, line width, line ending, `const` over `let`, etc.). That way we could really shorten that style guides to a minimal set of actual recommendations how to code, and not plain styling we can check automatically. That way I would feel way more welcome to adding this as a sign off into the list. I think if we add it currently, everyone will just mark that check, without actually knowing 50% of what they just signed off.

I don't think we should do the cleanup in this PR, but I would like to put not having this checkmark in here until we cleaned up up for discussion.

Reply:
Looking through the rules I think we could even add a couple of more eslint/tslint rules in places for some of the guidelines, that we even don't have covered yet (e.g. no use of `arguments`, use of `===` over `==`, etc.)

Reply:
I think that's fair. I'll remove it. At the end of the day this is me just being mad no one follows the sass and js guidelines that are set up. Putting an item in here doesn't force that, but linters probably can.